resource-manager/src/main/java/org/ihtsdo/otf/resourcemanager/ResourceConfiguration.java

```diff
@@ -94,13 +97,23 @@ public String toString() {
 
 		private String bucketName;
 		private String path;
+		private String accessKey;
+		private String secretKey;
+		private String region;
 
 		public Cloud() {
 		}
 
-		public Cloud(String bucketName, String path) {
+		public Cloud(final String bucketName,
+					 final String path,
+					 final String accessKey,
```

Comment:
We prefer to use environment variables for AWS credentials because when the application is run within AWS we don't need to set them, it just works. Not having the credentials in config on disk is more convenient and arguably more secure. Outside of AWS environment variables can be set in a startup script. [See docs for more info](https://docs.aws.amazon.com/sdk-for-java/v1/developer-guide/credentials.html).

Reply:
Fixed.